boards/v2/arm/mps3/board.yml

```diff
@@ -0,0 +1,7 @@
+board:
+  name: mps3
+  vendor: arm
```

Comment:
a suggestion here: why don't we re-use `vendor-prefixes.txt` (from dt-bindings), and guess the vendor name based on board location? I feel like this field is a bit redundant. So to summarize, boards/v2/$vendor_prefix/$board_name, if you want to obtain $vendor_name, just query vendor-prefixes.txt

Reply:
(not necessary to re-use vendor-prefixes, as there are many board vendors that do not produce the actual socs, but share the idea, ie, a database of prefixes/names)

Reply:
> why don't we re-use vendor-prefixes.txt (from dt-bindings),

are all vendors of boards having an entry there ?
if a vendor just creates a hw variant of an upstream board there may not need to be any reason for extra vendor prefixes.

For example: `bbc_microbit` doesn't seem to have an entry in vendor prefixes.

> and guess the vendor name based on board location?

no guessing / extraction of info from folder structure, it doesn't scale.
Although we intend to structure under vendor, the model itself has no such requirement and oot users with custom board may not see a reason for a vendor folder, as all boards in their board root is their own.

The idea with model v2 is exactly to not enforce a given structure.
Note, i'm very much in favor of guidelines on board structure for zephyr repo.

There some discussion on this in beginning of this PR.

Reply:
> not necessary to re-use vendor-prefixes, as there are many board vendors that do not produce the actual socs, but share the idea, ie, a database of prefixes/names

seems you noticed the same while I was writing my answer.

With the flexibility of organizing boards in new model, what would be the purpose / benefit of such file / db ?
If for build system / ci / twister, then we could extend `list_board.py` to easily produce such file for re-use between tests and thereby reduce overhead.

Reply:
I'd make `vendor` a vendor prefix, checked against a database file. Reasons include: avoid typos, obtain readable name from a single source, etc.

Reply:
I think this would make sense as an enhancement followup.